entry/vexflow-petaluma.ts

```diff
@@ -16,5 +16,7 @@ loadCustom();
 Flow.setMusicFont('Petaluma', 'Custom');
 loadTextFonts();
 
+// Re-export all exports from index.ts.
 export * from '../src/index';
-export default Vex;
+// Also collect all exports into a default export for CJS projects.
```

Comment:
Vex is no longer exprted, please remove line 7:
`import { Vex } from '../src/vex';`

Reply:
Thanks! Done.